enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/context/InteractiveQuestionProvider.tsx

```diff
@@ -26,6 +28,19 @@ export const InteractiveQuestionContext = createContext<
 
 const DEFAULT_OPTIONS = {};
 
+const FILTER_MODEL_MAP: Record<EntityTypeFilterKeys, DataPickerValue["model"]> =
```

Comment:
I've added this to provide some the user-facing terms to the user while still using our terms internally

Reply:
nit: the name of the const doesn't really give me the direction of the mapping